typescript/vscode-ext/packages/language-server/src/lib/baml_project_manager.ts

```diff
@@ -295,6 +295,10 @@ class Project {
 
     return runtime.list_functions()
   }
+  list_testcases(): BamlWasm.WasmTestCase[] {
+    let runtime = this.runtime()
```

Comment:
what happens if the runtime isnt defined like when theres a syntax error? Do we get a language server crash?

Reply:
Yeah you're right, should throw an error